src/utils/helpers.ts

```diff
@@ -13,3 +14,85 @@ export {
     clearObject,
     all
 }
+
+export const extractNumberFromStyle = (value: string) => Number(value.replace('px', ''))
+
+export const getPaddingValues = (padding?: string | number, paddingLeft?: string | number, paddingRight?: string | number) => {
+    if (!padding && !paddingLeft && !paddingRight) {
+        return null
+    }
+
+    if (!padding && paddingLeft && paddingRight) {
```

Comment:
i would create zeroPadding = {
  top: 0,
  right: 0,
  bottom: 0,
  left: 0
}
and then just merge it with gicen values:
return {
 ...zeroPadding,
left: extractPadding(0
}